AutomatedTesting/Gem/PythonTests/assetpipeline/asset_processor_tests/asset_processor_batch_tests.py

```diff
@@ -352,6 +350,9 @@ def run_ap_debug_skip_atom_output(asset_processor):
 
         shutil.copyfile(data_for_rename, file_to_rename)
 
+        # Clear the cache folder
+        asset_processor.delete_temp_cache()
```

Comment:
won't this alter the behavior? I think the test is specifically trying to make sure the naming collision is resolved cleanly, but if the cache is modified between runs, won't that impact how the test runs?

Reply:
Yeah that's a good point, I figured changing this could help as the processing order could make the test flaky. (If B.fbx processed before A.fbx) - sometimes one of them wouldn't reprocess because it wasn't needed - I could change the verification at the end to ensure all the expected product assets are present in cache instead of ensuring 2 assets processed.

Reply:
We have a command line parameter to run asset jobs in a very specific order, --reprocessFileList
That might help control the order this runs the second time.